src/logger.js

```diff
@@ -154,24 +156,39 @@ export function Logger({ url, prefix, logLevel = DEFAULT_LOG_LEVEL, transport =
             }
 
             if (amplitudeApiKey) {
-                transport({
-                    method:  'POST',
-                    url:     AMPLITUDE_URL,
-                    headers: {
-                        'content-type': 'application/json'
-                    },
-                    json: {
-                        api_key: amplitudeApiKey,
-                        events:  tracking.map((payload : Payload) => {
-                            // $FlowFixMe
-                            return {
-                                event_type:       payload.transition_name || 'event',
-                                event_properties: payload,
-                                ...payload
-                            };
-                        })
-                    }
-                }).catch(noop);
+                const data = {
+                    api_key: amplitudeApiKey,
+                    events:  tracking.map((payload : Payload) => {
+                        // $FlowFixMe
+                        return {
+                            event_type:       payload.transition_name || 'event',
+                            event_properties: payload,
+                            ...payload
+                        };
+                    })
+                };
+                if (window.navigator.sendBeacon(url, data)) {
+                    res = true;
```

Comment:
Should be no need to set `res` here. Only necessary if it's a promise we need to wait for

Reply:
Was heading towards being able to capture the response and saw it was set with request.  Wasn't sure why it was on one and not the other since we return a ZalgoPromise<void>.